lottie/src/main/java/com/airbnb/lottie/utils/LottieLogger.java

```diff
@@ -0,0 +1,14 @@
+package com.airbnb.lottie.utils;
+
+// Give ability to integrators to provide
```

Comment:
nit: the correct javadoc syntax is:
```
/**
 * 
 */
```
The way you have it, the IDE won't show it as the documentation.

Reply:
Will do